### PR TITLE
saved job svc test

### PR DESCRIPTION
**Pull request by Dhanrajnath**

### CODE REVIEW

**Comment by Dhanrajnath** [comment]
Hi @Thota-Aakanksha,
can you please ignore target folder and review

**Comment by Dhanrajnath** [comment]
https://sonarcloud.io/summary/overall?id=Dhanrajnath_backendbootcamp

**Comment by Thota-Aakanksha** [comment]
> Hi @Thota-Aakanksha, can you please ignore target folder and review

you can add the target folder in gitignore ?

File: src/main/java/com/greencommute/backend/controller/UserController.java [hunk 1]
```diff
@@ -79,9 +84,13 @@ public ResponseEntity<ResponseDto> saveJobsForUser(@PathVariable(value = "id") i
...
 
     @DeleteMapping("/{id}/savedJobs")
-    public ResponseEntity<ResponseDto> deleteSavedJobsOfUser(@PathVariable(value="id") int id, @RequestBody Map<String,Integer> reqPayload) {
+    public ResponseEntity<ResponseDto> deleteSavedJobsOfUser(@PathVariable(value="id") int id, @RequestBody Map<String,Integer> reqPayload)  throws DataNotFoundException  {
         final String jobId = "jobId";
-        savedJobService.deleteSavedJobs(id,reqPayload.get(jobId));
+        SavedJobs savedJob = savedJobsJpa.findByUserAndJobId(id, reqPayload.get(jobId));
+        if(savedJob == null){
+            throw  new DataNotFoundException("No saved job found with user id and job id");
+        }
```
Comment (Thota-Aakanksha) [inline src/main/java/com/greencommute/backend/controller/UserController.java:92]: did you test this in postman, is it working ?

**Review by Thota-Aakanksha — APPROVED** [review]

File: src/main/java/com/greencommute/backend/controller/UserController.java (hunk 1 above)
Comment (Dhanrajnath) [reply src/main/java/com/greencommute/backend/controller/UserController.java:92]: Screenshot from 2022-05-24 10-12-16

Comment (Dhanrajnath) [reply src/main/java/com/greencommute/backend/controller/UserController.java:92]: it's working @Thota-Aakanksha

Comment (Thota-Aakanksha) [reply src/main/java/com/greencommute/backend/controller/UserController.java:92]: > it's working @Thota-Aakanksha

ok

**Pull request merged**